app/src/main/java/org/novinger/jason/demoreader/datamodels/Article.java

```diff
@@ -0,0 +1,67 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 4/27/16.
+ */
+public class Article {
+    private String id;
```

Comment:
Ah, interesting. I think I misunderstood the `gson` docs here that I needed to name the member variables the same as the labels found in the JSON objects. Will look into this.